lib/ngx/errlog.md

```diff
@@ -210,6 +211,28 @@ clear the table yourself before feeding it into the `errlog.get_logs` function.
 
 [Back to TOC](#table-of-contents)
 
+get_sys_filtering_level
+-----------------------
+**syntax:** *log_level = log_module.get_sys_filtering_level()*
+
+**context:** *any*
+
+Return system default filtering level as an integer. Later the return value
+could be used as Nginx log level constant. For example:
```

Comment:
If I may, I think this would read better as well:
```
Return the system's default filtering level as an integer. Later, the returned 
value could be used as an Nginx log level constant. For example:
```